index.js

```diff
@@ -1082,15 +1176,28 @@ PersonaClient.prototype.error = function (message) {
     this.log(ERROR, message);
 };
 
+/**
+ * @param req Request||null
+ * @returns {*}
+ */
+PersonaClient.prototype.getXRequestId = function(req) {
+    if (_.isObject(req) && _.has(req,"header") && _.isFunction(req.header) && _.isString(req.header('X-Request-Id'))) {
```

Comment:
You can drop the `_.has` check - `_.isFunction` will return false if `req.header` is undefined/null.